src/compiler/scala/tools/nsc/transform/SpecializeTypes.scala

```diff
@@ -315,10 +326,14 @@ abstract class SpecializeTypes extends InfoTransform with TypingTransformers {
       if (sym.isClass) env.keySet
       else specializedTypeVars(sym).intersect(env.keySet)
     )
+    specializedName(sym.name, tvars, env)
+  }
+
+  private def specializedName(name: Name, tvars: immutable.Set[Symbol], env: TypeEnv): TermName = {
```

Comment:
IT does seem unnecessary (albeit not harmful) now, perhaps I had another caller during an earlier iteration of the patch.